test/array_queue_test.rb

```diff
@@ -3,15 +3,15 @@
 describe Adrian::ArrayQueue do
   it 'should allow construction with an array' do
     q = Adrian::ArrayQueue.new([1,2,3])
-    q.pop.value.must_equal 1
-    q.pop.value.must_equal 2
-    q.pop.value.must_equal 3
-    q.pop.must_be_nil
+    _(q.pop.value).must_equal 1
```

Comment:
What is this `_` syntax?

Reply:
This is a style specific to minitest https://minitest.rubystyle.guide/#global-expectations. I started by running all of the tests for the gem and saw some deprecation warnings, so I fixed them first. You can recreate the deprecation warnings by pulling main and running the tests.

Reply:
It actually looks like there are way more deprecations to be fixed.

Reply:
I mistakenly thought that `ruby test/*` was running every file in the test dir. It was only running the alphabetically first test. There are more deprecations to fix that are not relevant to the PR so I am undoing this for now.